Resources/Prototypes/Roles/Jobs/Engineering/senior_engineer.yml

```
@@ -22,6 +22,9 @@
   - Engineering
   - External
   - Atmospherics
+  special:
+  - !type:AddImplantSpecial
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)